frontend/src/components/ManageSchema.vue

```diff
@@ -57,21 +70,58 @@
         </v-tabs>
         <v-tabs-items v-model="tab">
           <v-tab-item transition="false" value="schema-attributes">
-            <v-card flat class="mt-2">
+            <v-card flat class="mt-2" v-show="!isEdit">
               <v-list-item
                 v-for="attribute in schema.schemaAttributeNames"
                 :key="attribute.id"
               >
                 <p class="grey--text text--darken-2 font-weight-medium">
                   {{ attribute }}
                   <v-icon
+                    :key="checkBoxGroup"
                     x-small
-                    v-if="attribute === schema.defaultAttributeName"
+                    v-show="isDefaultAttribute(attribute)"
                     >$vuetify.icons.asterisk</v-icon
                   >
                 </p>
               </v-list-item>
             </v-card>
+            <v-form v-show="isEdit">
+              <v-card flat class="mt-2">
+                <v-row>
+                  <v-col cols="8" class="py-0"> </v-col>
+                  <v-col cols="2" class="py-0"
+                    ><p class="grey--text">
+                      {{ $t("component.createSchema.headersColumn.isDefault") }}
+                    </p></v-col
+                  >
+                </v-row>
+                <v-row
+                  v-for="(attr, index) in schema.schemaAttributeNames"
+                  :key="attr.id"
+                >
+                  <v-col cols="8" class="py-0">
+                    <v-list-item>
+                      <p class="grey--text text--darken-2 font-weight-medium">
+                        {{ attr }}
+                        <v-icon x-small v-show="isDefaultAttribute(attr)"
+                          >$vuetify.icons.asterisk</v-icon
+                        >
+                      </p>
+                    </v-list-item>
+                  </v-col>
+                  <v-col cols="2" class="py-0">
+                    <v-checkbox
+                      v-model="checkBoxGroup"
+                      :value="index + 1"
+                      outlined
+                      dense
+                      style="margin-top: 4px; padding-top: 4px"
```

Comment:
This is not the vuetify way, see: https://vuetifyjs.com/en/styles/spacing/#how-it-works

Reply:
copy pasted this from CreateSchema.vue, will change it there too: tag "pt-1 mp-1"